### DISCUSSION TITLE
Are we only facing this issue with testcase history?

### DISCUSSION THREAD

**Discussion by SohanTirpude** [allure-framework/allure2#2407] [Bug Reports] [answered]
Hello all,

I am seeing same date time for different test case runs. This same issue(#1680) was mentioned in this forum in year 2022  but no one provided any comments on that.

<img width="1792" alt="310169134-197b6c3d-9087-4591-bcd2-f9e247a80966" src="https://github.com/allure-framework/allure2/assets/13250447/b8c58c1c-3314-4892-b262-34601e5510ca">

Are you all not seeing this issue at all in your Allure report UI? Or we are only ones?

@baev @epszaw @eroshenkoam @sseliverstov @vovsemenv Can you please look into this issue? As this is very critical for my project work.

**Comment by baev** [comment]
Could you please update Allure to the latest version ([2.27.0](https://github.com/allure-framework/allure2/releases/tag/2.27.0) as for today) and check again? 

(I can see you're using the old version because of the report logo — it was changed in 2.22.2)

**Comment by SohanTirpude** [comment]
Hello @baev,

Actually I am using the docker image provided by @fescobar from here: https://github.com/fescobar/allure-docker-service and it looks like it is not updated since more than 1 year.

@fescobar can you please provide the updated image with the latest version of Allure?

**Reply by fescobar** [reply, marked as answer]
I will update the allure version this week.

**Comment by baev** [comment]
@SohanTirpude, mate, could you please consider not opening issues in multiple repositories? 😃 Discussions are the best way to ask for help. Thanks!
